Sources/TSCBasic/FileSystem.swift

```diff
@@ -890,7 +951,7 @@ public class RerootedFileSystemView: FileSystem {
     }
 
     public func removeFileTree(_ path: AbsolutePath) throws {
-        try underlyingFileSystem.removeFileTree(path)
+        try underlyingFileSystem.removeFileTree(formUnderlyingPath(path))
```

Comment:
This seemed to be a bug

Reply:
I agree, that seems to have a bug all along.  Good catch!